### PR TITLE
feat: Make application stable

### LINKED ISSUE #487: Run the wasm in the broswer

### PR DESCRIPTION
**Pull request by ablax**
# Description

- What does this PR do?
Make the application stable enough to run it in the web and have almost everything ready for protocols implementation

Fixes https://github.com/LimeChain/Fruzhin/issues/487

## Checklist:
- [x] I have read the [contributing guidelines](https://github.com/LimeChain/Fruzhin/blob/dev/CONTRIBUTING.md).
- [x] My PR title matches the [Conventional Commits spec](https://www.conventionalcommits.org/).
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: build.gradle.kts [hunk 1]
```diff
@@ -20,20 +20,9 @@ dependencies {
     compileOnly("org.projectlombok:lombok:1.18.34")
     implementation("org.projectlombok:lombok:1.18.34")
 
-    implementation(fileTree(mapOf("dir" to "libs", "include" to listOf("*.jar"))))
+    implementation("org.teavm:teavm-jso-apis:0.10.0")
```
Comment (Zurcusa) [inline build.gradle.kts:23]: are both of these required separately?

File: build.gradle.kts [hunk 2]
```diff
@@ -42,9 +31,9 @@ teavm.js {
...
-//teavm {
-//    js {
-//        sourceMap.set(true)
-//        debugInformation.set(true)
-//    }
-//}
\ No newline at end of file
+teavm {
+    js {
+        sourceMap.set(true)
```
Comment (Zurcusa) [inline build.gradle.kts:37]: couldn't we merge this with teavm.js above?

File: src/main/java/com/limechain/network/Network.java [hunk 3]
```diff
@@ -9,6 +9,7 @@
 import com.limechain.sync.warpsync.WarpSyncState;
 import lombok.Getter;
 import lombok.extern.java.Log;
+import org.teavm.jso.JSBody;
```
Comment (Zurcusa) [inline src/main/java/com/limechain/network/Network.java]: Is this used?

File: src/main/java/com/limechain/sync/warpsync/WarpSyncMachine.java [hunk 4]
```diff
@@ -62,10 +62,11 @@ public boolean isSyncing() {
     public void start() {
         if (this.chainService.getChainSpec().getLightSyncState() != null) {
             LightSyncState initState = LightSyncState.decode(this.chainService.getChainSpec().getLightSyncState());
-//            if (this.syncState.getLastFinalizedBlockNumber()
-//                        .compareTo(initState.getFinalizedBlockHeader().getBlockNumber()) < 0) {
-//                this.syncState.setLightSyncState(initState);
-//            }
+            System.out.println(initState);
```
Comment (Zurcusa) [inline src/main/java/com/limechain/sync/warpsync/WarpSyncMachine.java]: Did you maybe forget this from debugging?

File: src/main/webapp/index.html [hunk 5]
```diff
@@ -19,9 +19,17 @@
...
+<body>
 <script>
-    let start = async () => {
+    var libp = undefined;
+
+    var getPeerId = () => {
+        return libp?.peerId;
+    }
+
+    var start = async (bootnodes) => {
```
Comment (Zurcusa) [inline src/main/webapp/index.html:30]: Where do we populate the bootnodes list?

**Review by Zurcusa — APPROVED** [review]

File: build.gradle.kts (hunk 1 above)
Comment (ablax) [reply build.gradle.kts:23]: seems like teavm-jso-apis includes teavm-jso

File: build.gradle.kts (hunk 2 above)
Comment (ablax) [reply build.gradle.kts:37]: The sourceMap and debugInformation will be removed later on

File: src/main/webapp/index.html (hunk 5 above)
Comment (ablax) [reply src/main/webapp/index.html:30]: It's fetched from chainspec

**Pull request merged**